types/brcodePayment/brcodePayment.d.ts

```diff
@@ -188,6 +190,27 @@ declare module 'starkbank' {
          */
         function update(id: string, params?: {status: string, user?: Project | Organization | null}): Promise<BrcodePayment>;
 
+        export class Rule {
+            /**
+             *
+             * BrcodePayment.Rule object
+             *
+             * @description The BrcodePayment.Rule object modify the behavior of BrcodePayment objects created in your Workspace.
```

Comment:
Mesmo comentário lá de cima que é bom deixar a mesma docstring para o BrcodePayment.Rule do Python.
The BrcodePayment.Rule object modifies the behavior of Transfer objects when passed as an argument upon their creation.